src/ui/public/vislib/visualizations/gauges/simple.js

```diff
@@ -104,7 +104,21 @@ export function SimpleGaugeProvider() {
       const tooltip = this.gaugeChart.tooltip;
       const isTooltip = this.gaugeChart.handler.visConfig.get('addTooltip');
       const yFieldFormatter = this.gaugeChart.handler.data.get('yAxisFormatter');
-      const fontSize = this.gaugeChart.handler.visConfig.get('gauge.style.fontSize');
+
+      let fontSize;
+      let deprecatedFontSize;
+      try {
+        deprecatedFontSize = this.gaugeChart.handler.visConfig.get('fontSize');
+        if (!isNaN(parseInt(this.gaugeChart.handler.visConfig.get('fontSize')))) {
```

Comment:
maybe check for deprecatedFontSize first and then use the variable try to parse it's value

`if (deprecatedFontSize && !isNaN(parseInt(deprecatedFontSize))) {`